src/com/facebook/buck/swift/SwiftCompile.java

```diff
@@ -147,6 +149,15 @@ private SwiftCompileStep makeCompileStep() {
       compilerCommand.add(
           "-import-objc-header",
           getResolver().getRelativePath(bridgingHeader.get()).toString());
+
+      // bridging header needs exported headers for imports
+      Path headerPath = CxxDescriptionEnhancer.getHeaderSymlinkTreePath(
+          getProjectFilesystem(),
+          BuildTargets.unflavored(getBuildTarget()),
```

Comment:
@nguyentruongtho me too. I'm testing stripping `SwiftLibraryDescription.SUPPORTED_FLAVORS`, `AppleDebugFormat` and `SwiftLibraryDescription.Type` now.

I was looking for a method that returns header tree for a given target but haven't found one.